ocaml/forkexecd/lib/fe_systemctl.mli

```diff
@@ -22,6 +22,34 @@ type status = {
 (** key -> value map *)
 type 'a string_map = 'a Map.Make(String).t
 
+val set_properties :
+     ?env:string string_map
+  -> ?properties:(string * string list) list
+  -> service:string
+  -> unit
+  -> unit
+(** [set_properties ?env ?properties ~service ()] sets properties on systemd
+      [service].
+
+  @param env Environment variables to set
+
+  @param properties systemd service properties to override.
+    [Some value] overrides the property.
+    [None] removes any override for the property, setting back to default
+
+  @param service the systemd service to change. Changes take effect on next
+  restart.
+
+*)
+
+val start_templated : template:string -> instance:string -> unit
+(** [start_templated ~template ~instance] starts [template@instance.service]
+  instance of the [template@.service] systemd template unit.
```

Comment:
It's problably worth mentioning here that the service files is created at this point, and it is deleted when stopping the service, which may not always be expected as these are not transient

Reply:
For start the service file is expected to exist already, I'll add this to the docstring.
On stop it indeed deletes the service file, but only if it was transient (a non-transient service won't be found in the transient path, so the unlink_safe will be a no-op). I'll document that too.